frontend/garo-language-explorer/style.css

```
@@ -0,0 +1,284 @@
+}
+
+.garo-play-btn:hover {
+    transform: translateY(-2px);
+    box-shadow: 0 4px 12px rgba(255, 153, 51, 0.3);
+}
+
+.garo-play-btn.playing {
+    animation: garoPulse 1.5s infinite;
+}
+
+@keyframes garoPulse {
```

Stylelint (17.14.0)

[error] Expected keyframe name "garoPulse" to be kebab-case (keyframes-name-pattern)

(keyframes-name-pattern)